engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/WorldSetupScreen.java

```diff
@@ -67,22 +69,36 @@
     private ModuleEnvironment environment;
     private Context context;
     private WorldConfigurator oldWorldConfig;
+    private Name camelCase;
```

Comment:
More "huh" at the "camel" here - at least it makes sense to some degree that you're involving camelCase the capitalization pattern, but that doesn't tell the user anything about what this variable is for. Name by its purpose not its formatting.